src/classes/Commands.ts

```diff
@@ -321,6 +325,94 @@ export = class Commands {
         );
     }
 
+    private messageCommand(steamID: SteamID, message: string): void {
+        const isAdmin = this.bot.isAdmin(steamID);
+        const parts = message.split(' ');
+
+        if (process.env.DISABLE_MESSAGES === 'true') {
+            if (isAdmin) {
+                this.bot.sendMessage(
+                    steamID,
+                    'The message command is disabled. Enable it in the config with `DISABLE_MESSAGES=false`.'
+                );
+            } else {
+                this.bot.sendMessage(steamID, 'The owner has disabled messages.');
+            }
+            return;
+        }
+
+        const adminDetails = this.bot.friends.getFriend(steamID);
+
+        if (isAdmin) {
+            if (parts.length < 3) {
+                this.bot.sendMessage(
+                    steamID,
+                    'Your syntax is wrong. Here\'s an example: "!message 76561198120070906 Hi"'
+                );
+                return;
+            }
+
+            const recipient = parts[1];
+
+            const recipientSteamID = new SteamID(recipient);
+
+            if (!recipientSteamID.isValid()) {
+                this.bot.sendMessage(steamID, '"' + recipient + '" is not a valid steamid.');
+                return;
+            } else if (!this.bot.friends.isFriend(recipientSteamID)) {
+                this.bot.sendMessage(steamID, 'I am not friends with the user.');
+                return;
+            }
+
+            const recipentDetails = this.bot.friends.getFriend(recipientSteamID);
+
+            const reply = message.substr(message.toLowerCase().indexOf(recipient) + 18);
+
+            // Send message to recipient
+            this.bot.sendMessage(
+                recipient,
+                'Message from ' + (adminDetails ? adminDetails.player_name : 'admin') + ': ' + reply
```

Comment:
Which is better? 'admin' or 'owner'? 🤔

Reply:
Admin is better because that's what it is called in the config